backend/src/v5/middleware/dataConverter/inputs/users.js

```diff
@@ -216,14 +216,11 @@ const generateSignUpSchema = (isSSO) => {
 
 	return captchaEnabled
 		? schema.test('check-captcha', 'Invalid captcha', async (body) => {
-			const checkCaptcha = post(config.captcha.validateUrl, {
-				secret: config.captcha.secretKey,
-				response: body.captcha,
-			});
-
+			const url = `${config.captcha.validateUrl}?secret=${config.captcha.secretKey}&response=${body.captcha}`;
```

Comment:
It would be clearer to do it like before, instead of appending all the parameters in the query string. You can still do this with axios https://axios-http.com/docs/req_config

Reply:
cause then we wont be using the webReuqests class

Reply:
> cause then we wont be using the webReuqests class

Shoudl still be using WebRequest class,it's just a abstraction to axios?